pkg/schema/load/dashboard.go

```diff
@@ -21,10 +23,16 @@ var panelSubpath = cue.MakePath(cue.Def("#Panel"))
 
 func defaultOverlay(p BaseLoadPaths) (map[string]load.Source, error) {
 	overlay := make(map[string]load.Source)
-	if err := toOverlay("/", p.BaseCueFS, overlay); err != nil {
+	currentDir, err := os.Getwd()
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	if err := toOverlay(currentDir, p.BaseCueFS, overlay); err != nil {
```

Comment:
in the toOverlay, according to the comments, cue overlay should be absolute path, that is why instead of changing it to toOverlay(os.pathseperator, p.BaseCueFS, overlay), i am putting currentDir instead. probably we can use environement variable to put it at home instead? not sure